architecture/additional_concepts/authorization.adoc

```diff
@@ -0,0 +1,194 @@
+authorization:
+
+[cols="1,7"]
+|===
+
+|[[identity]]*Identity* |Both the user name and list of groups the user belongs to.
+
+|[[action]]*Action* a|The action being performed. In most cases, this consists of:
+
+[horizontal]
+Project:: The link:../../dev_guide/projects.html[project] being accessed.
+Verb:: Can be `get`, `list`, `create`, `update`, or others.
+Resource:: The API endpoint being accessed.
+
+|*Bindings* |The full list of link:#bindings[bindings].
+
+|===
+
+OpenShift evaluates authorizations using the following steps:
+
+. The identity and the project-scoped action is used to find all bindings that
+apply to the user or their groups.
+. Bindings are used to locate all the roles that apply.
+. Roles are used to find all the rules that apply.
+. The action is checked against each rule to find a match.
+. If no matching rule is found, the action is then denied by default.
+
+== Cluster Policy and Local Policy
+There are two levels of authorization policy:
+
+[cols="1,4"]
+|===
+
+|*Cluster policy* |link:#roles[Roles] and bindings that are applicable across
+all projects. Roles that exist in the cluster policy are considered _cluster
+roles_. Cluster bindings can only reference cluster roles.
+
+|*Local policy* |link:#roles[Roles] and bindings that are scoped to a given
+project. Roles that exist only in a local policy are considered _local roles_.
+Local bindings can reference both cluster and local roles.
+
+|===
+
+This two-level hierarchy allows re-usability over multiple projects with cluster
+policy while allowing controlled customization inside of individual projects
+with local policy.
+
+During evaluation, both the cluster bindings and the local bindings are used.
+For example:
+
+. Cluster-wide "allow" rules are checked.
+. Locally-bound "allow" rules are checked.
+. Deny by default.
+
+== Roles
+Roles are collections of policy link:#rules-def[rules], which are sets of
+permitted verbs that can be performed on a set of resources. OpenShift includes
+a set of default roles that can be added to users and groups in the
+link:#cluster-policy-and-local-policy[cluster policy] or in a
+link:#cluster-policy-and-local-policy[local policy].
+
+[cols="1,4",options="header"]
+|===
+
+|Default Role |Description
+
+|*admin* |A project manager. If used in a
+link:#cluster-policy-and-local-policy[local binding], an *admin* user will have
+rights to view any resource in the project and modify any resource in the
+project except for role creation and quota.  If the *cluster-admin* wants to
+allow an *admin* to modify roles, the *cluster-admin* must create a
+project-scoped `*Policy*` object using JSON.
+
+|*basic-user* |A user that can get basic information about projects and users.
+
+|*cluster-admin* |A super-user that can perform any action in any project. Only
+applicable within the cluster policy and has no effect when granted to a user or
+group within a local policy.
```

Comment:
Not entirely true.  When granted to a user within a local policy, they will have full control over quota and roles. (every action on every resource in the project)